package.json

```diff
@@ -19,16 +19,17 @@
     "coverage-report": "npm run build && npm run nobuild:coverage-report",
     "coverage-html": "npm run build && npm run nobuild:coverage-html",
     "coverage": "npm run build && npm run nobuild:coverage",
+    "format": "npm run prettier -- --write",
+    "format:ci": "npm run prettier -- --check",
     "integration": "npm run build && npm run nobuild:integration",
     "nobuild:coverage-report": "nyc report --reporter=lcov",
     "nobuild:coverage-html": "nyc report --reporter=html",
     "nobuild:coverage": "nyc --check-coverage --branches 90 --functions 90 --lines 90 mocha",
     "nobuild:integration": "mocha --timeout 50000 test/integration/",
-    "nobuild:standard": "standard ts_src/**/*.ts",
     "nobuild:unit": "mocha",
     "prepare": "npm run build",
-    "standard": "npm run build && npm run nobuild:standard",
-    "test": "npm run build && npm run nobuild:standard && npm run nobuild:coverage",
+    "prettier": "prettier ts_src/*.ts ts_src/**/*.ts --ignore-path ./.prettierignore",
+    "test": "npm run build && npm run nobuild:coverage",
```

Comment:
`npm test` should also warn the user if their code is non-standard, so we should add `format:ci` here as well.

Reply:
done
https://github.com/bitcoinjs/bitcoinjs-lib/pull/1343/commits/763cb8b92312b817b657489b34d1dc3398932502